docs/articles/documentation/test-api/obtaining-data-from-the-client.md

```diff
@@ -176,6 +177,58 @@ test('My Test', async t => {
 });
 ```
 
+## Import Functions to be Used as Client Function Dependencies
+
+Assume you have a JS file `utils.js` with a function you need to use as a client function dependency in your test file.
+
+**utils.js**
+
+```js
+export function getDocumentURI() {
+    return document.documentURI;
+}
+```
+
+You would probably want to use the `import` statement to import this function but this will not always work.
```

Comment:
Yep, just show the right way to import CF's deps.

Reply:
According to what @AndreyBelym  said, you cannot use it if you pass it as the dependency *under the same name*. I assume that if you specify a different name, it will work. Am I wrong?

Reply:
okay (I didn't see @AndreyBelym's reply)

Reply:
> Yep, just show the right way to import CF's deps.

I thinks that we should say `don't use import` explicitly.

Because, well... Do you remember me asking what is special about the way we import functions?

https://github.com/DevExpress/testcafe/issues/2346#issuecomment-383896279

The thing is that this bare example doesn't suggest that we shouldn't use `import`.

(If I use `import` in my code, face problems with passing the imported function to the `ClientFunction` and find this example... I wouldn't think that the reason is `import`. Even if I see the difference between my code and the example, I may scan it through thinking "okay, here's import, here's require, there's no difference...")

Reply:
Although... maybe I'm wrong and a simple advice to use `require` is enough...